src/mesh/abstract_tree.jl

```diff
@@ -266,7 +290,7 @@ end
 # Refine given cells without rebalancing tree.
 #
 # Note: After a call to this method the tree may be unbalanced!
-function refine_unbalanced!(t::AbstractTree, cell_ids) end
+# function refine_unbalanced!(t::AbstractTree, cell_ids) end
```

Comment:
Why is this commented?

Reply:
You defined an empty function such that each `AbstractTree` inherits a `refine_unbalanced!(t::AbstractTree, cell_ids)` that does and returns `nothing`.